jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -51,6 +51,7 @@
         - github-pull-request:
             # This is the Jenkins GHPRB plugin ID, not the actual github token.
             auth-id: 'f8e31bc1-9abb-460a-a2ca-9c4aae3ca4e8'
+            only-trigger-phrase: {only-trigger-phrase}
```

Comment:
you might need to put this in single quotes.

you definitely will need to set `only-trigger-phrase` explicitly to false for all of the other builds.